src/Voron/Data/BTrees/TreeMutableState.cs

```diff
@@ -114,14 +130,21 @@ public void RecordFreedPage(TreePage p, int num)
             }
         }
 
+        public void CopyTo(TreeRootHeader* header)
```

Comment:
How expensive is this that we need to go to all this trouble?

Reply:
The JIT will emit 4 assembler instructions vs 2 per field.